melodies_monet/plots/aircraftplots.py

```diff
@@ -222,7 +228,167 @@ def add_yax2_altitude(ax, pairdf, altitude_yax2, text_kwargs, vmin_y2, vmax_y2):
 
     return ax
 
-                              
+
+
+
+###NEW curtain plot qzr++  (NEW CURTAIN model plot with model overlay, shared x-axis 
+def make_curtain_plot(time, altitude, model_data_2d, obs_pressure, pairdf, mod_var, obs_var, grp_dict, vmin=None, vmax=None, cmin=None, cmax=None, plot_dict=None, outname='plot', domain_type=None, domain_name=None, obs_label_config=None, text_dict=None, debug=False):
+    """
+    Generates a curtain plot comparing model data with obs across altitude (Pressure, right now) over time,
+    with the ability to customize the appearance through a configuration dictionary.
+    ##Two Subplots: 1) model data contourf plot with model scatter overlay, 
+    2) another for observation data using scatter plot. 
+    This layout ensures that both datasets can be analyzed without visual interference from each other.
+    Shared X-Axis: The time axis is shared between the two plots for better comparison.
+    Titles and Labels: Each subplot has a title specific to the data it displays.
+
+    Parameters
+    ----------
+    time : numpy.ndarray
+        Array of time points, expected to be numerical values suitable for plotting.
+    altitude : numpy.ndarray
+        Array of altitude points.
+    model_data_2d : numpy.ndarray
+        2D array of cleaned and interpolated model data.
+    obs_pressure : numpy.ndarray
+        Array of pressure points corresponding to observations.
+    pairdf : pandas.DataFrame
+        Paired DataFrame containing model and observation data for scatter plots.
+    mod_var : str
+        Model variable name for labeling.
+    obs_var : str
+        Observation variable name for labeling.
+    grp_dict : dict
+        Plot configuration options including aesthetics and normalization parameters.
+    vmin : float
+        Min value to use on y-axis.
+    vmax : float
+        Max value to use on y-axis.
+    cmin : float or None
+        Minimum value for color normalization, if applicable.
+    cmax : float or None
+        Maximum value for color normalization, if applicable.
+    plot_dict : dictionary
+        Dictionary containing information about plotting for each pair
+        (e.g., color, linestyle, markerstyle).
+    outname : str
+        File location and name of plot.
+    domain_type : str
+        Type of domain being plotted (e.g., 'region', 'global').
+    domain_name : str
+        Name of the domain being plotted.
+    obs_label_config : dict
+        Configuration dictionary for observation labels.
+    text_dict : dict
+        Dictionary containing text properties (fontsize, fontweight, etc.).
+    debug : bool
+        Whether to plot interactively (True) or not (False). Flag for submitting jobs to supercomputer turn off interactive mode.
+
+    Returns
+    -------
+    None
+        Saves the generated plot to a file.
+    """
+
+    if vmin is not None and vmax is not None:
+        plot_dict['ylim'] = [vmin, vmax]
+    
```

Comment:
The vmin and vmax values from your yaml file are not linked to the vmin and vmax values used here and this code is not used later on either. So likely should delete this line. And fix this connection like specified above.